tox.ini

```
@@ -17,8 +18,9 @@ deps =
     django22: Django>=2.2,<2.3
     django30: Django>=3.0,<3.1
     django31: Django>=3.1,<3.2
+    django32: Django~=3.2.0
```

```suggestion
    django32: Django>=3.2,<3.3
```
To be similar to the 2.2 example? Might cover any future patch releases

---

The [`~=`](https://www.python.org/dev/peps/pep-0440/#compatible-release) operator means compatible release. It is equivalent to `>=3.2,<3.3`.
Not feeling strongly about specifying one way or another.

---

I tend to side with @sarahboyce here, the `>=3.2,<3.3` is easier to understand for a reader who doesn't spend their time in pip requirement files :D